src/v2/view-builder/views/AdminConsent.js

```diff
@@ -0,0 +1,82 @@
+import { loc, View, Model } from 'okta';
+import BaseView from '../internals/BaseView';
+import BaseForm from '../internals/BaseForm';
+import ScopeList from '../../../views/admin-consent/ScopeList';
+import consentLogoHeaderTemplate from '../../../views/shared/templates/consentLogoHeaderTemplate';
+
+
+const Header = View.extend({
```

Comment:
put this in a new file `ConsentViewHeader.js`

Reply:
https://github.com/okta/okta-signin-widget/pull/1637/commits/5c38fe6dbc14fc42c30a7aededad0f7c937ff8d3